### ISSUE TITLE
Trying to convert YAML to JSON returns error "'extension.previewAsJason' not found"

### ISSUE THREAD

**Issue by traughber** [hilleer/vscode-yaml-plus-json#54]
**Describe the bug**
When I try to convert YAML to JSON returns error "'extension.previewAsJason' not found". 

**To Reproduce**
1. Highlight JSON content
2. Choose 'YAML+JSON: Preview As YAML' from command pallet.
3. See error "'extension.previewAsJason' not found". 

**Screenshots**
<img width="277" alt="CleanShot 2022-06-29 at 11 17 26@2x" src="https://user-images.githubusercontent.com/2132814/176511595-f3b52801-6f10-4306-af87-532b9a96359c.png">

**(please complete the following information):**
 - OS: MacOS 12.4
 - Version 1.10.0
 - Vscode version 1.68.1
 - Extension version 1.10.0

**Comment by traughber** [comment]
FYI, if I actually save the file as a JSON file and right-click on the file name and choose "Convert to YAML" then it works. Seems that just highlighting the text and choosing it from the pallet is what's not working. I'm happy to do the former as a work around.

**Comment by hilleer** [comment]
Alright. In the workspace you use, are there already some json and/or yaml files? Vscode extension uses this `activationEvents` and if neither of these are true, then it will actually not activate: https://github.com/hilleer/vscode-yaml-plus-json/blob/3f7f64b1077b78535c850bcdad0a2ab4fb0c1fec/package.json#L28-L40.

Maybe I need to look into optimising this.

**Comment by hilleer** [comment]
Can you try to do the following:

1. Click `cmd + j`, select `output` and select `Log (Extension host)
   <img width="1073" alt="Screenshot 2022-06-30 at 08 54 44" src="https://user-images.githubusercontent.com/9430821/176612386-513232d7-6288-497a-bd80-ed5bd9bb9b18.png">
2. Clear the output
   <img width="192" alt="Screenshot 2022-06-30 at 08 57 18" src="https://user-images.githubusercontent.com/9430821/176612689-c9c55ee5-5c9a-4fec-b134-584d92ef788d.png">
3. Reload vscode with the reload command
4. Click `cmd + f` and search for `hilleer.yaml-plus-json`.
   <img width="1078" alt="Screenshot 2022-06-30 at 08 58 30" src="https://user-images.githubusercontent.com/9430821/176612914-302eeb85-16fd-4cf7-b7c6-5585d6397e75.png">

Do you find anything like this?

**Comment by traughber** [comment]
My OUTPUT window was empty to begin with.
<img width="801" alt="CleanShot 2022-06-30 at 08 59 18@2x" src="https://user-images.githubusercontent.com/2132814/176724159-70e01a91-653c-42a3-a150-4430ca5d35b6.png">

**Comment by hilleer** [comment]
That's odd. And you've selected `Log (Extension host)` in the right side? What happens in this tab, if you try to use any command or feature of the extension?

**Comment by knight7024** [comment]
I had a same issue but it works after below behaviors:
1. Reload Window
2. Choose 'Convert selection to YAML' from command palette
3. Ignore error message
4. Choose 'YAML+JSON: Preview As YAML' from command palette

There're log from Extension Host.
```
[2022-07-29 02:55:18.490] [exthost] [info] ExtensionService#_doActivateExtension hilleer.yaml-plus-json, startup: false, activationEvent: 'onCommand:extension.jsonSelectionToYaml'
[2022-07-29 02:55:24.792] [exthost] [info] ExtensionService#_doActivateExtension vscode.configuration-editing, startup: false, activationEvent: 'onLanguage:json'
[2022-07-29 02:55:24.799] [exthost] [info] ExtensionService#_doActivateExtension vscode.extension-editing, startup: false, activationEvent: 'onLanguage:json'
[2022-07-29 02:55:24.810] [exthost] [info] ExtensionService#_doActivateExtension vscode.json-language-features, startup: false, activationEvent: 'onLanguage:json'
[2022-07-29 02:55:24.843] [exthost] [info] ExtensionService#_doActivateExtension vscode.npm, startup: false, activationEvent: 'onLanguage:json'
```

**Comment by hilleer** [comment]
Hi @knight7024 

Thanks for providing this info. Will have a look into activation events with this to see if there are some possible improvements to be made.

**Comment by hilleer** [comment]
Hi @traughber 

I have published a new version `1.11.0` of the extension that includes additional activation events. Can you please try to see if you can still reproduce this issue, after you have updated to latest version (remember to reload vscode afterwards)?

**Comment by hilleer** [comment]
Hi @traughber 

Have you had a chance to check if the newest version resolves the issue(s) you had?
